salt/modules/pkgng.py

```diff
@@ -51,9 +51,12 @@
 def __virtual__():
     '''
     Load as 'pkg' on FreeBSD 10 and greater.
+    Load as 'pkg' on DragonFly BSD.
     Load as 'pkg' on FreeBSD 9 when config option
     ``providers:pkg`` is set to 'pkgng'.
     '''
+    if __grains__['kernel'] == 'DragonFly':
+        return __virtualname__    
```

Comment:
Could you please remove the trailing whitespace here before we get this merged? Thanks.